compiler-core/src/erlang.rs

```diff
@@ -512,6 +512,71 @@ fn tuple<'a>(elems: impl IntoIterator<Item = Document<'a>>) -> Document<'a> {
         .group()
 }
 
+fn const_string_concatenate_bit_array<'a>(
+    elems: impl IntoIterator<Item = Document<'a>>,
+) -> Document<'a> {
+    join(elems, break_(",", ", "))
+        // We want to align the utf8 string parts of the bit array
+        // <<"a"/utf8,
+        //   "b"/utf8>>
+        // instead of
+        // <<"a"/utf8,
+        // "b"/utf8>>
+        .set_nesting(INDENT + 2)
```

Comment:
This will be wrong if the indent level is supposed to be anything other than 1!

For example:

```gleam
const x = [
  <<
    ...,
    ...,
  >>
]
```

I don't mind so much about the exact format used, but the indentation should increase as there's more nesting

Reply:
oh right, I pushed a change with a test case for a long concatenation in a list.
Let me know what you think, it doesn't exactly has the format that you specified before but the indentation should be good now.

I think to render like that:
```erl
main() ->
    <<"a"/utf8,
      "b"/utf8,
      "c"/utf8,
      ...
```
would require changes in the nesting logic or a special case I am not sure, I could look into it if you want

Reply:
Great job @DeviousStoat!! I think I can help with the formatting here. The code could look something like this (I haven't tested it but this should at least get you closer to a working solution 😁)
```rs
fn const_string_concatenate_bit_array<'a>(
    elems: impl IntoIterator<Item = Document<'a>>,
) -> Document<'a> {
  let elems = elems.collect_vec();
  if elems.is_empty {
    return "<<>>".to_doc();
  }
  
  let open = "<<".to_doc().append(break_("", "").nest(INDENT));
  let close = break_("", "").append(">>".to_doc());
  
  join(elems, break_(", ", ",").nest(INDENT))
    .surround(open, close)
    .group()
}
```

The look will be something like this if the bit array is split:
```erl
<<
  "wibble"/utf8,
  "wobble"/utf8
>>
```